.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,32 @@
+name: Bug report
+description: Create a new issue reporting a bug
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: A clear and concise description of what the bug is.
+    validations:
+      required: true
+  - type: textarea
+    id: reproduction-steps
+    attributes:
+      label: Reproduction steps
+      description: Steps to reproduce the behavior.
+      placeholder: 1. ...
+        2. ...
+        3. ...
```

Comment:
I think you might need to use `|` for the newlines here. See https://yaml.org/spec/1.2.2/#812-literal-style

Reply:
Added in fbaf003ea96a97d534ed5084e197825d5539df21 according to [GitHub's example](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms#yaml-issue-form-template).